### PR TITLE
Chunk rendering mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 1]
```diff
@@ -24,7 +24,8 @@ CLASS dkm net/minecraft/client/render/WorldRenderer
 	FIELD X chunkRendererList Ldkc;
 	FIELD Y renderDistance I
 	FIELD a DIRECTIONS [Lfa;
-	FIELD aa renderedEntities I
+	FIELD aa regularEntityCount I
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping:27]: Do we need the `regular`? I feel like it goes without saying

File: mappings/net/minecraft/client/render/WorldRenderer.mapping [hunk 2]
```diff
@@ -146,8 +155,10 @@ CLASS dkm net/minecraft/client/render/WorldRenderer
 	METHOD a setWorld (Ldhl;)V
 	METHOD a getOpenChunkFaces (Lev;)Ljava/util/Set;
 		ARG 1 pos
-	METHOD a getChunkRenderer (Lev;Ldne;Lfa;)Ldne;
+	METHOD a getNeighborChunkRenderer (Lev;Ldne;Lfa;)Ldne;
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/WorldRenderer.mapping]: `getAdjacentChunkRenderer`?

File: mappings/net/minecraft/client/render/WorldRenderer.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/client/render/WorldRenderer.mapping]: Maybe, but I just named it like that to match the text: `Regular: x/y Block: z/w`.

Comment (Gegy) [reply mappings/net/minecraft/client/render/WorldRenderer.mapping:27]: Ah, didn't realise it was named like that in-game. I suppose either works fine

Comment (Runemoro) [reply mappings/net/minecraft/client/render/WorldRenderer.mapping:27]: It excludes global entities (currently only the lightning, I think), which are normally considered entities in other parts of the code.

File: mappings/net/minecraft/client/render/chunk/ChunkRenderWorker.mapping [hunk 3]
```diff
@@ -1,4 +1,5 @@
 CLASS dnb net/minecraft/client/render/chunk/ChunkRenderWorker
+	CLASS net/minecraft/client/render/chunk/ChunkRenderWorker$1
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/chunk/ChunkRenderWorker.mapping]: I guess this is a bug?

Comment (Runemoro) [reply mappings/net/minecraft/client/render/chunk/ChunkRenderWorker.mapping]: Weird, fixed.

**Review by Gegy — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**